cfgov/v1/signals.py

```diff
@@ -53,25 +55,33 @@ def update_all_revisions(instance, attr):
 
 
 def unshare_all_revisions(sender, **kwargs):
-    from v1.wagtail_hooks import flush_akamai
     update_all_revisions(kwargs['instance'], 'shared')
-    flush_akamai()
 
 
 def unpublish_all_revisions(sender, **kwargs):
-    from v1.wagtail_hooks import flush_akamai
     update_all_revisions(kwargs['instance'], 'live')
-    flush_akamai()
 
 
 def configure_page_and_revision(sender, **kwargs):
-    from v1.wagtail_hooks import share, configure_page_revision, flush_akamai
-    share(page=kwargs['instance'], is_sharing=False, is_live=True)
+    from v1.wagtail_hooks import share, configure_page_revision
+    page = kwargs['instance']
+    share(page=page, is_sharing=False, is_live=True)
     configure_page_revision(
-        page=kwargs['instance'], is_sharing=False, is_live=True)
-    flush_akamai()
+        page=page, is_sharing=False, is_live=True)
+
+
+def flush_page(sender, **kwargs):
+    try:
+        kwargs['instance'].flush()
+    except AttributeError as e:
+        # This will happen when setting up the site locally
+        # (e.g. when running `tox`) since it creates a non-CFGOVPage Page
+        # which does not have `flush` defined
+        logger.info(e)
```

Comment:
Alternate shortcut:

```py
try:
    kwargs['instance'].flush()
except AttributeError:
    logger.exception('some message here')
```

This will automatically [include the stack trace](https://docs.python.org/2/library/logging.html#logging.Logger.exception) in the message.

That being said, is there an advantage in logging here, and not just ignoring?